pkg/controller/controller_instance_test.go

```diff
@@ -441,7 +499,7 @@ func TestReconcileInstance(t *testing.T) {
 
 	updatedInstance := assertUpdateStatus(t, actions[0], instance)
 	assertInstanceReadyTrue(t, updatedInstance)
-	assertInstanceDashboardURL(t, instance, testDashboardURL)
+	assertInstanceDashboardURL(t, updatedInstance, testDashboardURL)
```

Comment:
That was actually on rsc's want-to-haves for go in 2017: https://research.swtch.com/go2017

I agree it would be extremely convenient here.